apollo-compiler/src/test/graphql/com/example/fragments_with_type_condition/QueryExpected.java

```diff
@@ -1,29 +1,70 @@
 package com.example.fragments_with_type_condition;
 
+import com.apollostack.api.GraphQLQuery;
+import java.lang.Override;
+import java.lang.String;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
 import javax.annotation.Nullable;
 
-public interface Query {
-  @Nullable R2 r2();
+public final class Query implements GraphQLQuery<Query.Data> {
+  public static final String OPERATION_DEFINITION = "query Query {\n"
+      + "  r2: hero {\n"
+      + "    __typename\n"
+      + "    ...HumanDetails\n"
+      + "    ...DroidDetails\n"
+      + "  }\n"
+      + "  luke: hero {\n"
+      + "    __typename\n"
+      + "    ...HumanDetails\n"
+      + "    ...DroidDetails\n"
+      + "  }\n"
+      + "}";
 
-  @Nullable Luke luke();
+  public static final List<String> FRAGMENT_DEFINITIONS = Collections.unmodifiableList(Arrays.asList(
```

Comment:
doesn't sound like we need this. `fragmentDefinitions()` is not being called anywhere yet, right?